x-pack/test/api_integration/apis/telemetry/telemetry_local.ts

```diff
@@ -51,8 +51,18 @@ export default function ({ getService }: FtrProviderContext) {
     });
 
     it('should pass the schema validation', () => {
+      const excludedFromSchema = {
+        properties: {
+          kibana_config_usage: { type: 'pass_through' },
+        },
+      };
```

Comment:
I wonder if we should define this in `src/plugins/telemetry/schema/oss_root.json` so we don't need to change it in 3 files down the line. What do you think?